src/components/Tabber/PreviewModeSelector/PreviewModeSelector.components.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import Icon from '~components/Icon/Icon';
+
+import scss from './PreviewModeSelector.module.scss';
+
+import { PreviewModeProps } from './PreviewModeSelector.type';
+
+export function PreviewMode(props: PreviewModeProps): React.JSX.Element {
+  return (
+    <div key={props.item.id}>
```

Comment:
vai usar o id como key? E se tiver id igual?

Reply:
nesse caso, se o ID for igual, ele não faz alteração, então não vejo problema em usar o ID
![this](https://github.com/user-attachments/assets/4c31b8f8-af0a-4611-87b4-8beeb29c8c3b)